examples/aws/cloudformation/README.md

```diff
@@ -7,7 +7,12 @@
 We provide these CloudFormation templates as an example of how to setup HA Teleport in
 AWS using our AMIs. The current template will create a Highly available Teleport setup
 using DynamoDB and S3 to store logs. This setup will acquire a Let's Encrypt x509 certificate
-for you. 
+for you.
+
+If you are using the enterprise template please ensure that you have an enterprise license
```

Comment:
I would add a note, that these customers need to download it from the Gravitational Dashboard. https://dashboard.gravitational.com/web/